server/patches/server/level/ServerPlayer.patch

```diff
@@ -1,14 +1,32 @@
 --- a/net/minecraft/server/level/ServerPlayer.java
 +++ b/net/minecraft/server/level/ServerPlayer.java
-@@ -137,6 +137,7 @@
+@@ -137,6 +137,8 @@
  
  public class ServerPlayer extends Player implements ContainerListener {
  
 +    public java.util.OptionalInt viewDistance = java.util.OptionalInt.empty(); // Loom
++    public Component tabListName = null; // Loom
      private static final Logger LOGGER = LogManager.getLogger();
      public ServerGamePacketListenerImpl connection;
      public final MinecraftServer server;
-@@ -306,7 +307,7 @@
+@@ -192,6 +194,16 @@
+         this.textFilter = minecraftserver.createTextFilterForPlayer(this);
+     }
+ 
++    // Loom start
++    public void updateTabListName() {
```

Comment:
Same as above; this can easily be in the implementation. I don't really see a need to have it in the patch.